src/config/convert.js

```diff
@@ -443,9 +443,9 @@ const blackholeOutboundParser = (tag, params) => {
   }
 }
 
-const httpOutboundParser = (tag, params) => {
+const httpAndSocksOutboundParser = (protocol, tag, params) => {
   var ob = {
-    "protocol": "http",
```

Comment:
please make the field name explicit, I dont think depending on a var name is a good practice